cmd/sysl/cmd_catalog.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"github.com/anz-bank/sysl/pkg/catalog"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type catalogCmd struct {
+}
+
+func (p *catalogCmd) Name() string       { return "catalog" }
+func (p *catalogCmd) MaxSyslModule() int { return 1 }
+
+func (p *catalogCmd) Configure(app *kingpin.Application) *kingpin.CmdClause {
+	cmd := app.Command(p.Name(), "Generate ui catalog of applications and endpoints")
+	return cmd
+}
+
+func (p *catalogCmd) Execute(args ExecuteArgs) error {
+	args.Logger.Debugf("catalog: %+v", *p)
+	catalogServer := catalog.Server{
+		Port:    8080,
```

Comment:
Would it be possible to let user choose the port number? Maybe a `-p` flag, 8080 can be the default value but in case the port is used, user can specify other ports.